nncf/quantization/algorithms/weight_compression/mixed_precision.py

```diff
@@ -102,20 +102,13 @@ def apply(
             weight_param.compression_config = self._primary_config
             num_weights_in_4bit += weight_param.num_weights
 
-    @property
-    def available_backends(self) -> List[BackendType]:
-        return [BackendType.OPENVINO]
-
+    @abstractmethod
     def _set_backend_entity(self, model: TModel) -> None:
-        model_backend = get_backend(model)
-        if model_backend == BackendType.OPENVINO:
-            from nncf.quantization.algorithms.weight_compression.openvino_backend import OVMixedPrecisionAlgoBackend
```

Comment:
is `OVMixedPrecisionAlgoBackend` still needed?

Reply:
Thanks, typo.